README.md

```diff
@@ -23,15 +24,39 @@ Codefresh Runner (CI) has EBS optimized caching
 Codefresh Runner (CI) has no caching (Coming soon)
 
 ### Google
-./clouds/azure
+./clouds/google
 Codefresh Runner (CI) has no caching (Coming soon)
 
-
 ### Version Control
-(Coming soon)
 
 #### GitHub
-(Coming soon)
+To configure Version Control for GitOps Runtime you will need a Personal Access Token and define the following:
+
+(GitHub is only supported VCS right now.)
+
+- In your tfvars.json
+
+```json
+{
+  "create_isc": true,
+  "github_isc": true,
+  "github_api_token": "ghp_..",
+  "github_owner": ""
+}
+```
+
+
+
+- In gitops-runtime-values.yaml
+
+```yaml
+global:
+  runtime:
+    gitCredentials:
+      password: 
+        value: ghp_...
```

Comment:
Does it not expose the token? Would not it be better in a non-committed file like a tfvars one

Reply:
The issue with that would be that it is expected to then be present making this no longer an optional step.  

I also disliked this but to be honest... This is how Helm works today. :shipit: